src/main/java/seedu/address/model/person/Availability.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Candidate's availability.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDate(String)}
+ */
+public class Availability {
+
+    public static final String MESSAGE_CONSTRAINTS = "Availability should consists of a list of numbers from 1 to 7, "
+            + "separated by commas ','";
+    public static final String VALIDATION_REGEX = "^[1-7]{1}+(?:,[1-7]{1}+)*$";
```

Comment:
Nice use of regex!

Reply:
I agree @lzan98! Good job @LeeEnEn! Smart pick on using regex rather than iterating through an array.